scripts/launch.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-
 # Parse the spark default image from the oshinko CLI version info
 # For a version of oshinko-web that doesn't use the CLI, the value would just be set here
 SPARK_DEFAULT=$($(dirname $0)/oshinko version | sed -rn 's/Default spark image:\s(.*)/\1/p')
```

Comment:
This can't actually still work, can it?  The install of the CLI was removed.
We need a new mechanism to set the spark version.